src/Sql/Platform/IbmDb2/SelectDecorator.php

```diff
@@ -82,6 +104,23 @@ protected function processLimitOffset(PlatformInterface $platform, DriverInterfa
             return;
         }
 
+        if ($this->supportsLimitOffset) {
+            // Note: db2_prepare/db2_execute fails with positional parameters, for LIMIT & OFFSET
+            $limit = (int)$this->limit;
+            if (! $limit) {
```

Comment:
no space after the !

Reply:
most of zf code has space after `!`, I think already part of zf cs /cc @webimpress

Reply:
@mwillbanks 
@samsonasik is right. See zendframework/zend-coding-standard#1 - `Generic.Formatting.SpaceAfterNot`

Reply:
@mwillbanks 
Yes, and this is even part of current version of `zend-coding-standard` (1.0.0): https://github.com/zendframework/zend-coding-standard/blob/master/ruleset.xml#L12